tests/performance/ci_run.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -x -e
```

Comment:
I would recommend using `-o pipefail` and ideally `-u`, if possible.

Reply:
Thanks for the feedback.